content/applications/general/email_communication/email_servers.rst

```diff
@@ -1,222 +1,209 @@
 ====================================================
-Send and Receive Emails in Odoo with an Email Server
+Send and receive emails in Odoo with an email server
 ====================================================
 
-If you are a user of Odoo Online or Odoo.sh...
-==============================================
+Odoo Online or Odoo.sh users
+============================
 
-You have nothing to do! **Odoo sets up its own mail servers for your database.**
-Outgoing and incoming emails work out-of-the-box!
-
-Unless you plan to send large batches of mass mailing that could require the 
-use of an external mail server, simply enjoy your new Odoo database.
+Nothing needs to be done! **Odoo sets up its own mail servers for the database.** Outgoing and
+incoming emails work out-of-the-box!
 
+Unless an external mail server is required to send large batches of mass mailing, simply enjoy the
+new Odoo database.
```

Comment:
Suggestions around sentence structure to make this section clearer to understand (optional).
```suggestion
Since **Odoo sets up its own mail servers for the database**, outgoing and incoming emails already
work out-of-the-box. So for **Odoo Online** and **Odoo.sh** customers, nothing needs to be done!

Unless an external mail server is required to send large batches of mass mailing, simply use the
standard online Odoo database normally since it has already been preconfigured for email.
```